DEVELOPMENT.md

```diff
@@ -74,7 +74,6 @@ recommend adding them to your `.bashrc`):
 export GOPATH="$HOME/go"
 export PATH="${PATH}:${GOPATH}/bin"
 export KO_DOCKER_REPO='gcr.io/my-gcloud-project-name'
-export DOCKER_REPO_OVERRIDE="${KO_DOCKER_REPO}"
 export K8S_CLUSTER_OVERRIDE='my-k8s-cluster-name'
```

Comment:
You can remove this line as well.

Reply:
I delete all the K8S_* vars in this file. Please help to review it. Thanks!